application/backend/src/main/resources/apps/typerefinery/components/content/header/template.html

```diff
@@ -0,0 +1,54 @@
+<!--/*
+    Copyright (C) 2022 Dynamic Solutions
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/-->
+<template data-sly-template.header="${ @ model }">
+    <sly data-sly-test.image="${model.image}"></sly>
+    <sly data-sly-test.navigation="${model.navigation}"></sly>
+    <sly data-sly-test.cta="${model.cta}"></sly>
+
+    <sly data-sly-test="${image || navigation || cta}">
+        <nav class="hl-header" style="background: #001E3C; color: #FFF !important; margin-top:0rem">
+            <div class="hl-header__wrapper">
+                <div class="hl-header__top-bar">
+                    <sly data-sly-test="${image}" data-sly-use.template="/libs/howlite/components/image/template.html"
+                        data-sly-call="${template.simpleImage @ defaultImage=image.defaultImage, alt=image.alt, showLink=image.showLink, url=image.url, openInNewTab=image.openInNewTab, classes='hl-header__logo-image'}">
```

Comment:
Why are we using howlite?

Reply:
This is for the Logo thing.

We did not have any Image or Functional Button component as of now.

So Included both Prime Vue Navbar and Howlite thing.